app/models/user.rb

```diff
@@ -109,6 +109,9 @@ def before_add_method(role)
 
   has_many :point_transactions
 
+  has_one :about_user, dependent: :destroy
+  accepts_nested_attributes_for :about_user
```

Comment:
Can you remind me what this is for?

Reply:
The `accepts_nested_attributes_for`?

Reply:
Hmmm I can't seem to remember why did I put this thing...

Reply:
Think it should be safe to remove. I can't find usages of `accepts_nested_attributes_for` in the codebase anyway.

Reply:
👍